### ISSUE TITLE
Incorrect detekt.yaml path resolving for nested modules.

### ISSUE THREAD

**Issue by nikitin-da** [arturbosch/sonar-kotlin#51]
We have following project structure:

```
app
modules/module1
modules/module2
detekt.yaml
```

Setting `/detekt.yaml` as configuration path in Sonar/Kotlin/Detekt results in following error:

```
Provided path '/…/ca4c5a47ac213f623228/modules/detekt.yaml' does not exist
```

Because `tryFindDetektConfigurationFile` looks for file in module directory, then if not found looks on directory above which doesn't work if module is nested deeper.

https://github.com/arturbosch/sonar-kotlin/blob/30125207e3c45142d0f89b2515c8174fd80204a6/src/main/kotlin/io/gitlab/arturbosch/detekt/sonar/sensor/DetektConfiguration.kt#L49-L67

Ideal solution would be to support "absolute" path that starts with `/` and do something like:

```kotlin
if (path.startsWith("/")) {
  configFile = File(rootProjectPath, path)
} else {
  configFile = File(baseDir, path)
}
```

Where `rootProjectPath` is absolute path to the root of the project.

**Comment by arturbosch** [comment]
Yeah your write. I forgot to support absolute paths. As sonar treats each module as the `root` in a `Sensor` it is hard to find the config file so I wrote about this limitation in the readme.
Do you might to file a PR for this? :)
